pkg/ui/keymap/keymap.go

```diff
@@ -198,7 +198,7 @@ func DefaultKeyMap() *KeyMap {
 			"right",
 		),
 		key.WithHelp(
-			"→",
+			"→/l",
```

Comment:
Let's keep this one as is. Many Vim users are wired to use `l` and `h` subconsciously that we don't need to make it explicit in the help menu.

Reply:
Actually, nvm, this aligns more with the existing view.